### ISSUE TITLE
Marantz NR1710

### ISSUE THREAD

**Issue by MichaelHendriks** [hilvarenbeek/nl.marcovandenhout.marantz#31]
Can’t find the HEOS input.

**Comment by hilvarenbeek** [comment]
Hi, 
Unfortunately HEOS cannot be controlled in the same way. You will need to use the HEOS app. As for myself, I'm currently not actively developing this 'app' (new maintainers or pull requests welcome)

**Comment by MichaelHendriks** [comment]
It is correct what you indicate. But HEOS is a kind of input. Source “internet radio” does not work, because it's under HEOS.

**Comment by hilvarenbeek** [comment]
Ah, maybe it's just an input with a different name from what we already have in the app. If you know the technical input name, you could try sending a custom command starting with SI, for example SIDVD would set the input to a source named DVD. Unfortunately I don't have a device at hand to try it myself.

**Comment by MichaelHendriks** [comment]
I'll try, I'll let you know.

**Comment by MichaelHendriks** [comment]
The "USB" and the "HEOS" input are not working. The "Internet Radio" input works by selecting the "Net" input in the Homey app.
![marantzNR1710remote](https://user-images.githubusercontent.com/55698537/71023243-9ca60a80-2102-11ea-9b04-26390ce4eb6a.jpg)
Tell me which custom commands I can test that you use for the Homey app. I would like to help you integrate the commands of the Marantz NR1710 / NR1510
What does SI stand for?

**Comment by MichaelHendriks** [comment]
Got It! SI = Source Input?

**Comment by hilvarenbeek** [comment]
Yes, SI is the main zone select input. If you have separate zones, you can
also use e.g. Z2 to change inputs on that zone. Z2NET for example. But you
have to know the input names. And HEOS is a different beast, they have a
different protocol for HEOS.

Op di 17 dec. 2019 om 19:35 schreef MichaelHendriks <
notifications@github.com>:

**Comment by MichaelHendriks** [comment]
I think they just use CLI command lines for HEOS. "SINet" works for HEOS Internet Radio.

Or do you think it is the last used input that I used in HEOS. SINet is more like the HEOS button and not the Internet Radio button.
I have not yet tested what I describe.

**Comment by Patrick1610** [comment]
Hi @hilvarenbeek, I see that you know some stuff about custom commands. Where can I find more information about this? I would like to test some more with custom commands but wouldn't know where to start. Is there any documentation about this?

**Comment by hilvarenbeek** [comment]
Hi,

For some models Marantz themselves have published the codes in Excel sheets. Google for marantz IP-232 for example

Verstuurd vanaf mijn telefoon
